uSync.BackOffice/Hubs/uSyncHubRoutes.cs

```diff
@@ -28,7 +28,8 @@ public class uSyncHubRoutes : IAreaRoutes
             IRuntimeState runtimeState)
         {
             _runtimeState = runtimeState;
-            _umbracoPathSegment = globalSettings.Value.GetUmbracoMvcArea(hostingEnvironment);
+            // _umbracoPathSegment = globalSettings.Value.GetUmbracoMvcArea(hostingEnvironment);
+            _umbracoPathSegment = "__uSync__";
```

Comment:
@KevinJump Please make this configurable so it's easier to integrate in firewalls 🙂